mappings/net/minecraft/block/WorldModifyingBlock.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_8427 net/minecraft/block/WorldModifyingBlock
```

Comment:
A bit vague since many other blocks modify the world in some way. I can't think of a better name, though.

Reply:
Agree. It's not that good, but I can't think of a better term for placing or breaking